husky_directory/templates/full_results.html

```diff
@@ -2,35 +2,42 @@
 {% for scenario in search_result["scenarios"]%}
     {% if scenario['num_results'] > 0 %}
         <p>
-        <h4>{{scenario['description']}}</h4>
-        {# population: models.search.DirectoryQueryPopulationOutput #}
+        {# population: str
+         # results: models.search.DirectoryQueryPopulationOutput
+        #}
         {% for population, results in scenario['populations'].items() %}
-            {% for data in results['people'] %}
-                <h3>{{ data['name'] }}</h3>
-                <ul class="dir-listing">
-                    <li>{{ data['email'] }}</li>
-                    {% for contact_method, numbers in
-                       data['phone_contacts'].items() %}
-                        {% if numbers|length %}
-                            <li>{{ contact_method|singularize|titleize }}:
-                                {{ numbers|join(', ') }}</li>
+            {% if results['num_results'] > 0 %}
+                <h3 class='scenario-anchor'
+                    id="{{ population ~ '-' ~ scenario['description']|linkify }}">
+                    {{ population|externalize|titleize}}: {{ scenario['description']}}
+                </h3>
```

Comment:
I added this `if` block, and the `h3`; i changed the `data["name"]`  field to `h4`. 

The rest of the changes below here were to indent the block because of the new `if` and can be ignored.